nightfall-client/start-client

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash
+
+if [ -f client.env ]; then
+  grep -v '^#' client.env
+  # Export env vars
+  export $(grep -v '^#' client.env | xargs)
+else
+  echo "client.env does not exist. You need to define the client.env with the needed variables to run the client."
+  exit 1
+fi
+
+VOLUME_LIST=$(docker volume ls -q)
+FILE=
+
+usage()
+{
+  echo "Usage:"
+  echo "  -c or --client; for client service"
+  echo "  -d or --dev; to bind mount the filesystem and use it for development"
+}
+
+# select a Geth or Ganache client
+if [ -z "$1" ]; then
```

Comment:
What is this for?

Reply:
Removed. Expected at least 1 argument. With -c by default and after removal then it has no sense.